src/theme/index.ts

```diff
@@ -0,0 +1,44 @@
+import breakpoints from './breakpoints';
+import icons from './icons';
+
+import { extendTheme, ThemeOverride } from '@chakra-ui/react';
+
+export default extendTheme(<ThemeOverride>{
+  breakpoints,
+
+  colors: {
+    black: '#16161D',
```

Comment:
my ocd brain is tingling

```suggestion
    black: '#16161d',
```